Service invocation supports passing of query strings/fragments in URL (#4217)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

dapr invoke --app-id checkout --method checkout/100
```

> You can also append a query string or a fragment to the end of the URL and Dapr will pass it through unchanged. This means that if you need to pass some additional arguments in your service invocation that aren't part of a payload or the path, you can do so by appending a `?` to the end of the URL, followed by the key/value pairs separated by `=` signs and delimited by `&`. For example:

Reviewer: ```suggestion
#### Including a query string in the URL

You can also append a query string or a fragment to the end of the URL and Dapr will pass it through unchanged. This means that if you need to pass some additional arguments in your service invocation that aren't part of a payload or the path, you can do so by appending a `?` to the end of the URL, followed by the key/value pairs separated by `=` signs and delimited by `&`. For example:
```

> You can also append a query string or a fragment to the end of the URL and Dapr will pass it through unchanged. This means that if you need to passsome additional arguments in your service invocation that aren't part of a payload or the path, you can do so by appending a `?` to the end of the URL followed by the key/value pairs separated by `=` signs and delimited by `&` as in the following example:

Reviewer 2: ```suggestion
You can also append a query string or a fragment to the end of the URL and Dapr will pass it through unchanged. This means that if you need to pass some additional arguments in your service invocation that aren't part of a payload or the path, you can do so by appending a `?` to the end of the URL, followed by the key/value pairs separated by `=` signs and delimited by `&`. For example:
```

Author: It appears the DCO bot once again doesn't much care for the sign-off if there's a co-author listed as well.
